### DISCUSSION TITLE
Listbox autoFocus

### DISCUSSION THREAD

**Discussion by anubra266** [adobe/react-spectrum#3170] [Q&A]
I want to automatically focus the first item in my listbox, and when the collection changes, auto focus the new first item. I'm using the listboc in a combobox.
Thanks

**Comment by LFDanLu** [comment]
So you'll want to be careful with item focus in general for comboboxes. For our combobox, we ran into issues with NVDA not properly announcing what the user typed in input field if there was an item currently focused in the listbox hence why we don't autofocus items when the user types (if you are interested, you can read up more about this [here](https://react-spectrum.adobe.com/blog/building-a-combobox.html)).
